Content.Client/UserInterface/Controls/RadialContainer.cs

```diff
@@ -67,36 +67,86 @@ public RadialContainer()
     {
 
     }
-	
+
     protected override void Draw(DrawingHandleScreen handle)
     {
-		
-        const float baseRadius = 100f;
+		const float baseRadius = 100f;
         const float radiusIncrement = 5f;
 		
-        var children = ReserveSpaceForHiddenChildren ? Children : Children.Where(x => x.Visible);
-        var childCount = children.Count();
+        var children = ReserveSpaceForHiddenChildren
+            ? Children
+            : Children.Where(x => x.Visible);
+
+        // elements are children (buttons) and dividers, they all are spread evenly
+        var elementCount = children.Count() * 2;
 		
 		// Add padding from the center at higher child counts so they don't overlap.
-		Radius = baseRadius + (childCount * radiusIncrement);
+		Radius = baseRadius + (elementCount * radiusIncrement);
```

Comment:
This adjustment changes the overall area of the container, so there's a large disparity in size between menu layers that contain only a few elements and those that contain many. Consider having a set size for the sectors and only shifting the positions of the icons

On the topic of size, the amount of screen real estate that the sectors take up is quite large, consider reducing the outer radius as well, if it doesn't cause issues with the above

Reply:
i'll remove separators from radius increment - that makes sense, it was my failure to properly understand why it is there.
About size of sectors and 'free-space' - the point of this UX changes was to make it require less precision - so you can click it faster and not really aim, but make choice of direction. Size is just for that - most of games have this 'wheel' ui blur and grow in opacity from center to outer corners - i could not make it work here, sadly (tbf was not trying) but REAL size of radial menus were quite the same overall (if we remove blur and make controls constant color), i belive!
![image](https://github.com/user-attachments/assets/329cae48-6066-4fec-9c07-e47a796998bd)

Reply:
If there is some proposal about how to change numbers exactly - you are welcome to ask and i will try it - currently its radius * 2 for outer, radius /2 for inner

Reply:
Changed back to only use child count thanks to eoin. Kinda missed it initially. Yay!.
Multipliers were moved to properties - so can be easily changed.